manywheel/build_common.sh

```diff
@@ -387,10 +387,10 @@ for pkg in /$WHEELHOUSE_DIR/torch*linux*.whl /$LIBTORCH_HOUSE_DIR/libtorch*.zip;
     record_file=$(echo $(basename $pkg) | sed -e 's/-cp.*$/.dist-info\/RECORD/g')
     if [[ -e $record_file ]]; then
         echo "Generating new record file $record_file"
-        rm -f $record_file
+        : > "$record_file"
```

Comment:
why is ": > " better than "rm -f"?

Reply:
The file needs to be truncated / cleared, not deleted. The bug is specifically by the deletion, because you can see the logic is conditional on the existence of that file existing: https://github.com/pytorch/builder/blob/89c8ff725fabf6d82302a4010717cfff488d1a25/manywheel/build_common.sh#L268

An alternative way to write `: > "$record_file"` could be `truncate -s 0 "$record_file"`. However, I'm not sure on how prevalent that approach is. Redirects (> and >>) are quite common, so thats why I used that style.